src/scss/custom/components/_footer.scss

```diff
@@ -0,0 +1,164 @@
+footer {
```

Comment:
Yeah, actually now that we're adding non Bootstrap components here we should establish a namespace prefix for component classes. In mofo-ui it's `mui-`. 

`mofo-` seems fine to me. We should document it in the README as well.